### PR TITLE
AG-9212 - Split scatter and bubble series apart.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-9212

Splits `scatter` into `scatter` and `bubble` series types.

### CODE REVIEW

File: packages/ag-charts-community/src/chart/themes/chartTheme.ts [hunk 1]
```diff
@@ -399,6 +398,25 @@ export class ChartTheme {
...
+                marker: {
+                    ...ChartTheme.getCartesianSeriesMarkerDefaults(),
+                    maxSize: 30,
+                },
+                label: {
+                    enabled: false,
+                    fontStyle: undefined,
+                    fontWeight: undefined,
+                    fontSize: 12,
+                    fontFamily: DEFAULT_FONT_FAMILY as unknown as string,
```
Comment (olegat) [inline packages/ag-charts-community/src/chart/themes/chartTheme.ts]: `as unknown as string` seems redundant as of [19f5a4422](https://github.com/ag-grid/ag-charts/commit/19f5a4422542460507d9d3f0d6e3b03e1d1f7b5e)

Comment (alantreadway) [reply packages/ag-charts-community/src/chart/themes/chartTheme.ts]: Fixed - thanks!

File: packages/ag-charts-website/src/content/docs/scatter-series/index.mdoc [hunk 2]
```diff
@@ -35,6 +35,7 @@ A bubble chart is simply a scatter plot where each point has another associated
 To turn a scatter plot into a bubble chart you must provide the `sizeKey` that will be used to fetch the value that will determine the size of each bubble. For the example below we are using the following key configs:
```
Comment (manapeirov) [inline packages/ag-charts-website/src/content/docs/scatter-series/index.mdoc:35]: do we want to say you need to use the `bubble` series type?

**Review by manapeirov — APPROVED** [review]

File: packages/ag-charts-website/src/content/docs/scatter-series/index.mdoc (hunk 2 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/scatter-series/index.mdoc:35]: I'm not intending to address docs generally as part of this change - Rob is keen to get the implementation changes made and he'll take responsibility for docs aspects.

**Pull request merged**